src/features/delegation/components/DelegateButton.tsx

```diff
@@ -27,8 +31,15 @@ export function DelegateButton({ delegatee }: { delegatee: Delegatee }) {
       <div>
         <SolidButton
           className="btn-neutral w-full"
+          disabled={isValidator === true || isValidatorGroup === true}
           onClick={() => showTxModal()}
         >{`️🗳️ Delegate voting power`}</SolidButton>
+        {(isValidator || isValidatorGroup) && (
```

Comment:
UI guy hat on:

We have another red text that can also occur. but that one is a warning whereas this one is a hard NO. It is also theoretically possible for both to show and take up a lot of space. 

IDEA: 

dont disable here. rather allow delegate modal to open and then display then disable the button in the modal and dislay this explanation in the white blank space above button. 


<img width="461" alt="Screenshot 2025-01-10 at 2 20 03 PM" src="https://github.com/user-attachments/assets/27ee1180-fc3a-497c-9d0d-bd906f9fc9e3" />